cli/src/test/java/org/aya/cli/PlctReportTest.java

```diff
@@ -0,0 +1,15 @@
+// Copyright (c) 2020-2022 Tesla (Yinsen) Zhang.
+// Use of this source code is governed by the MIT license that can be found in the LICENSE.md file.
+package org.aya.cli;
+
+import org.aya.cli.plct.PLCTReport;
+import org.aya.cli.utils.MainArgs;
+import org.junit.jupiter.api.Test;
+
+public class PlctReportTest {
+  @Test public void generateReport() throws Exception {
+    var args = new MainArgs.PlctAction();
+    args.plctReport = true;
+    System.out.println(new PLCTReport().run(args));
```

Comment:
This test takes 7s to finish in my machine. And the whole tests take 11s.

Reply:
LMFAO

Reply:
I guess we should enabled it only when `"true".equals(System.getenv("CI"))`